drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -428,6 +446,66 @@
     outline: 2px dotted white;
 }
 
+@media screen and (max-width: 480px) {
+    .xblock--drag-and-drop .popup {
+        background-color: #fff;
+        border-radius: 10px;
+        text-align: center;
+        box-shadow: 0 0 100px rgba(0,0,0,0.4);
```

Comment:
Sorry I missed this during previous pass, but can you also add this rule here: `border: none`. It removes a 1px wide white border which is a bit distracting.

Reply:
Ans also: `max-height: 90vw;` to allow it to use more vertical space.